ide/nativeimage.api/src/org/netbeans/modules/nativeimage/api/debug/NIDebugger.java

```diff
@@ -165,6 +170,49 @@ public String getVersion() {
         return provider.getVersion();
     }
 
+    /**
+     * Get a list of locations for a given file path.
+     *
+     * @param filePath a file path
+     * @return list of locations, or <code>null</code> when there's no location
+     *         information about such file.
+     * @since 0.2
+     */
+    @CheckForNull
+    public List<Location> listLocations(String filePath) {
+        return provider.listLocations(filePath);
+    }
+
+    /**
+     * Get a list of functions of a given name.
+     *
+     * @param name a name pattern
+     * @param includeNondebug include also symbols from the symbol table
+     * @param maxResults maximum number of results
+     * @return list of source information and their symbols, or <code>null</code>
+     *         when there are no matching symbols.
+     * @since 0.2
+     */
+    @CheckForNull
+    public List<Pair<SourceInfo, List<Symbol>>> listFunctions(String name, boolean includeNondebug, int maxResults) {
```

Comment:
What's the difference between `name` and `filePath` at line 176?

Reply:
`name` is the name of the symbol. `filePath` is the path to the file the symbol is declared in.